app.test.js

```diff
@@ -127,4 +127,74 @@ describe("GET /api/articles", () => {
             expect(resBody).toBeSortedBy("created_at", {descending : true})
         })
     })
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("returns an error when given article id is not a number type", () => {
+        return request(app)
+        .get("/api/articles/notANumber/comments")
+        .expect(400)
+        .then(({body}) => {
+            expect(body.message).toBe("invalid id path")
+        })
+    })
+    
```

Comment:
Great test descriptions, really clear! Could improve them slightly by prefixing them with the error code you're expecting, eg
`400:returns an error when given article id is not a number type`